crates/fj-app/build.rs

```diff
@@ -0,0 +1,78 @@
+use std::{
+    path::PathBuf,
+    process::{Command, Output, Stdio},
+};
+
+fn main() {
+    println!("cargo:rustc-env=FJ_VERSION_STRING={}", version_string());
+}
+
+fn version_string() -> String {
+    let pkg_version = std::env::var("CARGO_PKG_VERSION").unwrap();
+    let commit = git_description();
+
+    let official_release = std::env::var("FJ_OFFICIAL_RELEASE").is_ok();
```

Comment:
Minor issue: If someone ever set `FJ_OFFICIAL_RELEASE=0`, this would be confusing.

Reply:
Good point.